themes/digital.gov/layouts/partials/core/guides/guide-page-head.html

```diff
@@ -1,31 +1,11 @@
-<header class="page-head page-head-{{- .Type -}}">
+<header class="page-head-{{- .Type -}}">
```

Comment:
Can we get rid of this header element? 

It's not necessary and it causing the prose styles to ignore the `h1`. This leads to redundant styling like we see in `_guide-page-head.scss:7`.